WooCommerce/Classes/Notifications/LocalNotification.swift

```diff
@@ -59,10 +62,10 @@ extension LocalNotification {
                       scenario: scenario,
                       actions: .init(category: .loginError, actions: [.contactSupport]))
         case .invalidPasswordFromWPComLogin, .invalidPasswordFromSiteAddressLogin:
```

Comment:
Super nit and probably unrelated: do we need to rename `invalidPasswordFromSiteAddressLogin` to `invalidPasswordFromSiteAddressWPComLogin` to differentiate with the site credential password?

Reply:
yea makes sense, renamed in https://github.com/woocommerce/woocommerce-ios/pull/7503/commits/654804bb379dd990dd14b808c60638f760de1837